### ISSUE TITLE
Jekyll build error

### ISSUE THREAD

**Issue by ilessing** [UCSBCarpentry/2020-05-29-UCSB-R#4]
I added a Github action to this repo.  The action does a test `jekyll build` and [the build](https://github.com/UCSBCarpentry/2020-05-29-UCSB-R/runs/633908969?check_suite_focus=true) failed with an error.

Here is the error:

>    GitHub Metadata: No GitHub API authentication could be found. Some fields may be missing or have incorrect data.
> jekyll 3.8.5 | Error:  "<USERNAME>/<PROJECT>" is invalid as a repository identifier. Use the user/repo (String) format, or the repository ID (Integer), or a hash containing :repo and :user keys.
> ##[error]Process completed with exit code 1.
>

**Comment by ilessing** [comment]
<img width="1552" alt="Screen-Shot-2020-04-30-at-12 16 12-PM" src="https://user-images.githubusercontent.com/275899/80750473-01279c00-8add-11ea-9250-f7f7359295c3.png">

**Comment by ilessing** [comment]
[this is where I found a hint on fixing the jekyll build](https://mmistakes.github.io/minimal-mistakes/docs/configuration/#site-repository).  It may be that putting the actual user name and repository in would cause the build to malfunction if you're building locally rather than on Github.

**Comment by torwhite** [comment]
Ian, I just saw that you requested my review. Approved, then github wanted me to merge the changes back in. I did that and it broke the build. Is that because you had already committed the changes so your pull request didn't actually need to be merged back in. Wondering before I just get rid of the last commit.

**Comment by ilessing** [comment]
Hi Torin
I made this pull request just try out the process of pull requests.  I think that any build errors you saw after the merge were un-related to the actual code changes we made.  I think the build error we saw under the Actions tab is due to a Ruby Gem install gone awry in Github's system.  I've see that same error on a couple of different repositories.
So.... I think you did just right in merging the pull request.
